### ISSUE TITLE
Stop putting background color on spaces

### ISSUE THREAD

**Issue by kballard** [sharkdp/hexyl#25]
If a byte's coloration includes a background color, the following space also gets the same background color.

The fix for this is trivial but any PR to fix it right now would conflict with #23.

**Comment by sharkdp** [comment]
None of the colorizations includes a background color, but yes, we can fix this.

**Comment by kballard** [comment]
Here's the offending code:

```rust
        let byte_str = format!("{:02x} ", b);
        write!(self.stdout, "{}", Byte(b).color().paint(byte_str))?;
```

The painted section includes the space.

That said, looking a bit further, it does appear that the specified colors are just RGB values and should just be painted as a foreground color. I'm not sure why I'm seeing background colors then. My terminal doesn't support arbitrary RGB colors, but one would expect `ansi_term` to be mapping the RGB colors to the nearest xterm256 palette color then.

**Comment by kballard** [comment]
Looks like `ansi_term` does not do that at all and is just outputting the escapes for arbitrary RGB colors. I guess this also explains why the colors look fairly badly-chosen on my terminal.

**Comment by kballard** [comment]
I guess this is already being discussed in #9.

**Comment by sharkdp** [comment]
I'm going to fix this soon... see #9.

**Comment by sharkdp** [comment]
Fixed in [v0.3.1](https://github.com/sharkdp/hexyl/releases/tag/v0.3.1).
